src/App.tsx

```diff
@@ -1,27 +1,55 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { GoodsList } from './GoodsList';
+import { Good } from './types/Good';
+import { getAll, get5First, getRed } from './api/goods';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+export const App: React.FC = () => {
+  const [goods, setGoods] = useState<Good[]>([]);
 
-export const App: React.FC = () => (
-  <div className="App">
-    <h1>Dynamic list of Goods</h1>
+  const allGoods = async () => {
```

Comment:
```suggestion
  const loadAllGoods = async () => {
```
It's better to name your functions according to these rules, so it's clear that it's a function
![image](https://user-images.githubusercontent.com/96409563/217629609-f846f85d-5d2a-465e-88e9-ec68d5983165.png)